public/index.html

```diff
@@ -1 +1 @@
="searchInput"></div><div class="moduleContainer"><div><div class="sectionTitle">Download</div><a href="./placeholder_largeimage.png" download="appIcon">Download Icon</a><br/><a id="downloadLink" download="DownloadViaLink.csv">In-Memory Download Link</a><br/><button id="downloadButton">In-Memory Download Button</button></div><div uploadfilesection><div class="sectionTitle">Upload</div><label for="image_uploads">Choose images to upload (PNG, JPG)</label> <input type="file" id="image_uploads" name="image_uploads" accept=".jpg, .jpeg, .png" multiple="multiple"><div class="fileUploadPreview"><p>No files currently selected for upload</p></div></div><div><div class="sectionTitle">Implicit Teams Deep Link</div><button onclick="enableImplicitExecuteDeepLink()">enable Implicit ExecuteDeepLink</button><div><a href="https://teams.microsoft.com/l/app/f46ad259-0fe5-4f12-872d-c737b174bcb4">https://teams.microsoft.com/_#/l/app/f46ad259-0fe5-4f12-872d-c737b174bcb4</a></div><div><a href="https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.planner/All" target="_blank">https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.planner/All</a></div><div><a href="https://microsoft.com/" target="_blank">https://microsoft.com/ (non teams deep link)</a></div></div><div><div class="sectionTitle">Print</div><button onclick="enablePrintCapability()">enablePrintCapability</button> <button onclick="printHandler()">Print this page</button></div><div><div class="sectionTitle">URIs</div><div><a href="sip:yudogan@microsoft.com">Yuri Dogandjiev SIP URI</a></div><div><a href="sip:yudogan@microsoft.com" target="_blank">Yuri Dogandjiev SIP URI (New Window)</a></div><div><a href="search-ms:query=Microsoft_Teams_Update.exe">Search protocol link</a></div><div><a href="search-ms:query=Microsoft_Teams_Update.exe" target="_blank">Search protocol link (New Window)</a></div><div><a href="msteams://teams.microsoft.com/l/entity/0d820ecd-def2-4297-adad-78056cde7c78/MyNotebook">OneNote app via msteams protocol</a></div><div><a href="msteams://teams.microsoft.com/l/entity/0d820ecd-def2-4297-adad-78056cde7c78/MyNotebook" target="_blank">OneNote app via msteams protocol (New Window)</a></div><div><a href="ms-teams://teams.microsoft.com/l/entity/0d820ecd-def2-4297-adad-78056cde7c78/MyNotebook">OneNote app via ms-teams protocol</a></div><div><a href="ms-teams://teams.microsoft.com/l/entity/0d820ecd-def2-4297-adad-78056cde7c78/MyNotebook" target="_blank">OneNote app via ms-teams protocol (New Window)</a></div></div><div><div class="sectionTitle">Re-navigation</div><div><a href="/page1.html">Page 1</a></div><div><a href="/page2.html">Page 2</a></div><div><a href="/scroll.html">Scroll</a></div><div><a href="/download.html">Download</a></div></div><div><div class="sectionTitle">External links</div><div><a href="https://www.bing.com" target="_blank">Bing via anchor tag</a></div><button onclick="window.open('https://www.bing.com')">Bing via window.open</button></div></div></body></html>
```

Comment:
Are these just formatting changes?

Reply:
I can't tell the difference, but this file shows in git change after yarn build